src/style/App.css

```diff
@@ -16,7 +16,7 @@
 }
 
 html {
-  scroll-behavior: smooth;
+  scroll-behavior: auto;
```

Comment:
It must be `auto`, otherwise, it creates a slow scroll movement issue
See https://github.com/dirtyredz/react-scroll-up-button/issues/14